Add API documentation to Konnect (#4633) (#4660)
Kong/docs.konghq.com: app/konnect/api/index.md

---
title: Konnect APIs
content-type: explanation
---

## Base URLs

There are two types of base URLs that are used in {{site.konnect_short_name}} APIs:

* Global: https://global.api.konghq.com 

* Region specific: https://{REGION_CODE}.api.konghq.com

The global endpoint is used to manage region-agnostic {{site.konnect_short_name}} entities that live in a global database. These APIs interact with entities that are not region specific and therefore do not have any regional boundaries. For example, identity management and IdP settings are all region-agnostic configurations applied to the organization as a whole.

The region-specific endpoints are used to manage {{site.konnect_short_name}} entities that live in a specific {{site.konnect_short_name}} region. These APIs interact with entities specific to a region and do not have the ability to cross regional boundaries. For example, runtime groups, service listings are all region-specific entities and therefore must use the region-specific endpoint.

> Identity Management API Docs

Reviewer: ```suggestion

```

> ## Authorization

Reviewer: ```suggestion
## Authentication
```

> The recommended method of authentication for {{site.konnect_short_name}} is Personal Access Token (PAT) which can be obtained from the personal access token page in the {{site.konnect_short_name}} dashboard. The PAT must be passed in the header of all requests, for example: 

Reviewer: ```suggestion
The recommended method of authentication for {{site.konnect_short_name}} is Personal Access Token (PAT), which can be obtained from the [personal access token page](https://cloud.konghq.com/global/tokens) in {{site.konnect_short_name}}. The PAT must be passed in the `Authorization` header of all requests, for example: 
```

Reviewer: suggestion: If you don't want to include a link to the Personal Access Tokens page, describe where to find it. I'm not sure what "the Konnect dashboard" is, and I doubt users will be, either.
